apps/parsley/src/utils/filterLogs/index.ts

```diff
@@ -64,10 +64,10 @@ const filterLogs = (options: FilterLogsParams): (number | number[])[] => {
     if (expandableRows) {
       // If the line doesn't match the filters, collapse it.
       const previousItem = arr[arr.length - 1];
-      if (isCollapsedRow(previousItem)) {
-        previousItem.push(idx);
+      if (isSkippedLinesRow(previousItem)) {
+        previousItem.lineEnd = idx + 1;
       } else {
-        arr.push([idx]);
+        arr.push({ lineEnd: idx + 1, lineStart: idx, rowType: "SkippedLines" });
```

Comment:
nit. I had a thought for an improvement on this to make it a little more readable. What if we converted this into a util method and we can use this everywhere we need to create a SkippedLines row?
```suggestion
        arr.push(newSkippedLineRow(idx, idx + 1));
```
This could also apply to section header rows
```ts
arr.push(newSectionHeaderRow("some function name"))
```
instead of 
```ts
arr.push({ functionName: "some function", rowType: "SectionHeader" })
```
What do you think?